nemo_skills/dataset/aalcr/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# settings that define how evaluation should be done by default (all can be changed from cmdline)
+PROMPT_CONFIG = "generic/default"
+DATASET_GROUP = "long-context"
+METRICS_TYPE = "answer-judgement"
```

Comment:
I don't think that's the right type. This one is used for comparing the judge performance itself. I think the right one would be to use the same setup as for HLE here https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/evaluation/metrics/map_metrics.py#L34. We should probably add a separate metric type for such cases of judge-only results in the future.

Reply:
I guess we'd need a new metric type here, added another comment about that

Reply:
Added a new metric aalcr here which also has detailed analysis. I prefer to have a more general metric type and move analysis elsewhere to keep it clean.